jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/ProgressAssert.java

```diff
@@ -19,7 +20,7 @@ public ProgressAssert width(Matcher<String> condition) {
         return this;
     }
     public ProgressAssert width(String width) {
-        return width(is(width));
+        return width(containsString(width));
```

Comment:
it's cannot catch the error when we are checking width(1) but actually there is 10 on site